src/planetary_system.jl

```diff
@@ -352,6 +352,340 @@ function generate_period_and_sizes_power_law(s::Star, sim_param::SimParam; num_p
 end
 
 
+
+
+
+using SpecialFunctions
```

Comment:
Maybe we should move purely mathematical functions to their own file.

Reply:
Sounds good. You can decide how to do this.

Reply:
Ok, I'll merge your PR first.